apps/condo/domains/ticket/schema/ExportIncidentsToExcelService.js

```diff
@@ -53,13 +55,11 @@ const classifiersToString = (classifiers = []) => {
 
     const renderProblems = problems ? ` » ${problems}` : ''
 
-    return `${place} » ${categories}${renderProblems}`
+    return `${categories}${renderProblems}`
 }
 
-const formatDate = (date, timeZone, format = 'DD.MM.YYYY HH:mm') => {
-    if (!date || !format) return EMPTY_SYMBOL
-    if (!timeZone) return dayjs(date).format(format)
-    return dayjs(date).tz(timeZone).format(format)
+const formatDate = (date, timeZone, format) => {
```

Comment:
please use default value like `format = DATE_FORMAT`

Reply:
Does it make sense? It is below overridden if empty